constraint/Makefile

```diff
@@ -109,3 +107,22 @@ generate-defaults: constraint-template-string-constant
 vendor:
 	go mod vendor
 	go mod tidy
+
+.PHONY: __test-image
+__test-image:
+	docker buildx build . \
+		-t constraint-test \
+		--load \
+		--build-arg KUSTOMIZE_VERSION=$(KUSTOMIZE_VERSION)
+
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
```

Comment:
If a user runs this in their source code directory, this will create a /bin folder. Do we want to put this in an easy-to-clean subdir (.output or similar)? Is the /bin directory part of .gitignore?

Reply:
looks like bin would be ignored, though I'd personally consider putting this in a .output directory to make it easy to wipe everything and start from zero.

Reply:
@maxsmythe `bin` is the default for kubebuilder v3. looks like we are using `.tmp` already (for linter) so i changed to `.tmp/bin`